packages/dag-history-component/src/components/State/index.tsx

```diff
@@ -125,30 +127,31 @@ export default class State extends React.PureComponent<IStateProps, IStateState>
     const marginLeftValue = successor ? 30 : 0;
 
     return (
-      <Motion style={{marginLeft: spring(marginLeftValue)}}>
-        {interpolatingStyle => (
-          <div
-            className={classnames('history-state', { successor })}
-            style={{
-              ...this.props.style,
-              backgroundColor,
-              marginLeft: interpolatingStyle.marginLeft
-            }}
-            onClick={e => handleClick(e)}
-          >
-            {continuation}
-            <div className="state-detail">
-              <div className={classnames('state-source', { active })}>
-                {source || ''}
-              </div>
-              <div className={classnames('state-name', { active })}>
-                {label || ''}
-              </div>
-            </div>
-            {bookmark}
+      <div
+        className={classnames('history-state', { successor })}
+        style={{
+          ...this.props.style,
+          backgroundColor,
+        }}
+        onClick={e => handleClick(e)}
+      >
+        <ReactCSSTransitionGroup
+          transitionName="continuation-dissolve"
```

Comment:
I don't like the name 'animation', because it should be known implicitly that it's an animation.